docs/MatrixOne/Develop/connect-mo/connect-to-matrixone-with-go.md

```diff
@@ -57,11 +57,63 @@ go version
 
 4. Execute this golang file in the command line terminal.
 
-    ```
+    ```bash
     > go run golang_connect_matrixone.go
     Database Connection Succeed
     ```
 
+## Using Gorm to connect to MatrixOne
+
+Gorm: The fantastic ORM library for Golang aims to be developer friendly.```gorm.io/gorm``` and ```gorm.io/driver/mysql``` will vbe used to make Go connect to MYSQL.
```

Comment:
Gorm is the fantastic ORM library for Golang and aims to be developer-friendly. `gorm.io/gorm` and `gorm.io/driver/mysql` will make Go connect to MYSQL.

Reply:
ok